pkg/repo/mysql/services.go

```diff
@@ -0,0 +1,92 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+func (r *Repo) SaveService(ctx context.Context, service *model.Service) error {
+	query := `INSERT INTO Service (ID, Name) VALUES (?, ?)
+	ON DUPLICATE KEY UPDATE Name = VALUES(Name)`
+
+	_, err := r.db.ExecContext(ctx, query, service.ID, service.Name)
+	if err != nil {
+		return fmt.Errorf("db: insert service: %w", err)
+	}
+
+	return nil
+}
+
+func (r *Repo) LoadService(ctx context.Context, serviceID uuid.UUID) (*model.Service, error) {
+	query := `SELECT ID, Name FROM Service WHERE ID = ?`
+	row := r.db.QueryRowContext(ctx, query, serviceID)
+
+	var service model.Service
+
+	err := row.Scan(&service.ID, &service.Name)
+
+	switch {
+	case errors.Is(err, sql.ErrNoRows):
+		return nil, repo.ErrNotFound
+	case err != nil:
+		return nil, fmt.Errorf("db: select service: %w", err)
+	}
+
+	return &service, nil
+}
+
+func (r *Repo) LoadServices(ctx context.Context) ([]model.Service, error) {
```

Comment:
Doesn't looks like a LoadServices function, more like GetServices.

Reply:
Can't say anything about that, but I also like idea of `Create, Delete, Get, List` more.
Those names that are now came from Horsts, personally don't know which are better

Reply:
Maybe LoadServices was the handler name not for db func?

Reply:
https://github.com/translate-agent/translate/issues/40#issue-1620982418

Reply:
In repo, `Load` and `Save` are alternatives to `Get` and `Create/Update`. They convey the same meaning. The aim is to avoid defining two repo methods to create or update an entity. 

In the past, I always used `Create` and `Update` but I feel we can get away with `Save` to simplify the repo interface.